README.md

```diff
@@ -4,6 +4,9 @@
 
 A collection of samples demonstrating how to use Spectrum Spatial for Big Data.
 
+### Support for Spark 3
+This sample supports till Spark 3.0.2. For version higher than Spark 3.1.x onwards, please contact precisely for any details.
```

Comment:
It should be under "Samples" heading and after  "A collection of samples demonstrating how to use Spectrum Spatial for Big Data.".

Also remove "For version higher than ..." line as it is including Precisely Tech support.

Reply:
I incorporated the advised change.